File: 240914/함수를 이용한 윤년 판별/tell-the-function-using-a-leap-year.py
def yoonyear(n):
    if n % 4 != 0:
        return False
    elif n % 100 != 0:
        return True
    elif n & 400 == 0:
        return True
    return False

y = int(input())
if yoonyear(y):
    print("true")
else:
    print("false")

File: 240914/함수를 이용한 윤년 판별/tell-the-function-using-a-leap-year.c
#include <stdio.h>
#include <stdbool.h>

bool yoonyear(int n) {
    if (n % 4 != 0) return false;
    else if (n % 100 != 0) return true;
    else if (n % 400 == 0) return true;
    return false;
}

int main() {
    int y;
    scanf("%d", &y);

    if (yoonyear(y)) {
        printf("true");
    }
    else {
        printf("false");
    }
    return 0;
}